### ISSUE TITLE
feature request: show recursive dependencies

### ISSUE THREAD

**Issue by SethTisue** [scalacenter/scaladex#281]
it would be useful to me if I could see the entire recursive dependency tree of a library, before I decide whether to add it to my project

currently the alternative is to add the project to my build and then use sbt-dependency-graph and try and sort out what's new because of the addition and what isn't. Scaladex could be a time-saver here

**Comment by MasseGuillaume** [comment]
This is not easy to implement. Do you have a solid use case?

**Comment by SethTisue** [comment]
it's just part of the due diligence I would do on literally any library right even in the very early stages of deciding whether to add that library to my project. as a responsible project maintainer I need to know everything that I'm putting on my classpath. if library is being added three dependency levels deep, I need to know about it and research the implications of adding it to my project. might it have a dependency that conflicts with an existing dependency I already have? might it have a dependency that uses reflection and therefore won't work in certain use scenarios? might it have a dependency that isn't actively maintained and therefore is a threat to my own project in the long term? might it have a dependency that doesn't work on Scala.js? etc etc etc etc

doing this kind of due diligence is something I consider a very basic part of running a well-managed software project.

there's no specific use scenario. I consider it much more fundamental than that. if I'm going to marry this library, I need to know its family.

**Comment by jvican** [comment]
I agree this would be super useful. I would personally use it on a daily basis.
